src/app/app.component.ts

```diff
@@ -55,11 +55,11 @@ export class AppComponent implements OnInit {
     do {
       const childrenRoutes = currentRoute.children;
       currentRoute = null;
-      childrenRoutes.forEach(route => {
+      childrenRoutes.forEach((route) => {
```

Comment:
Please configure your editor (VS Code) to not to add bracket around single param in arrow function. If you are using VSCode and installed prettier, you  can configure this in prettier extension settings.